apis/types/registry_service_config.go

```diff
@@ -41,7 +41,7 @@ type RegistryServiceConfig struct {
 	// > are in compliance with any terms that cover redistributing
 	// > nondistributable artifacts.
 	//
-	AllowNondistributableArtifactsCIDRs []string `json:"AllowNondistributableArtifactsCIDRs,omitempty"`
```

Comment:
Please revert this file when you `git commit`. We do not need this change. And the change done by swagger-0.12.0 is because swagger has a bug in itself.